addons/web/static/src/views/graph/graph_controller.js

```diff
@@ -31,11 +31,19 @@ export class GraphController extends Component {
     getContext() {
         // expand context object? change keys?
         const { measure, groupBy, mode } = this.model.metaData;
-        return {
+        const context = {
             graph_measure: measure,
             graph_mode: mode,
             graph_groupbys: groupBy.map((gb) => gb.spec),
         };
+        if (mode !== "pie") {
+            context.graph_order = this.model.metaData.order;
+            context.graph_stacked = this.model.metaData.stacked;
+        }
+        if (mode === "line") {
```

Comment:
you could put that block within the if (mode !== "pie") {

Reply:
is it working well when order === null?

Reply:
> is it working well when order === null?
Yes, it seems to work. In the `_getProcessedDataPoints` function of `graph_model.js`, we check that the order is different than `null` before ordering the records :smiley:

Reply:
Hello @Polymorphe57 :wave: ,

When I restore a favorite and I inspect the request fetching the graph data, I see that the request includes the new keys (i.e: `graph_order`, `graph_stacked`, `graph_cumulated`) in the context of the request but the `web_read_group` python function completely ignores those parameters.

When I edit and resend the request from the network tab of my browser, I don't get any error from the server by setting null values for those parameters. It seems like the server converts `null` to `None` (see the `self._context` in `web_read_group`). The new parameters will only be used on the client side (unless I missed something ?). So, it should be safe :smiley:

We can maybe omit those values on the `_loadDataPoints` function of `graph_model.js` ?